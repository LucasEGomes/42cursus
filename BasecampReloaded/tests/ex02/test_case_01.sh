#!/bin/sh
mkdir -p temp
touch temp/\#delete_me{0..5}\#
touch temp/do_not_delete_me{0..5}\#
touch temp/\#do_not_delete_me{0..5}
touch temp/do_not_delete_me{0..5}
cp $ROOT_DIR/ex02/clean temp/
cd temp
bash clean | sort > ../test_case_01.user
cd ..
diff test_case_01.expected test_case_01.user > test_case_01.output
RESULT=$(find temp | wc -l)
if [ -z "$(cat test_case_01.output)" ] && [ "$RESULT" -eq 20 ];
then
    echo -n "O" >> $TEST_SUMMARY
else
    echo -n "X" >> $TEST_SUMMARY
fi
rm -rf temp/
