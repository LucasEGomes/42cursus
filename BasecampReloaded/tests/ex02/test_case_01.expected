./#delete_me0#
./#delete_me1#
./#delete_me2#
./#delete_me3#
./#delete_me4#
./#delete_me5#
